reframe/frontend/cli.py

```diff
@@ -232,8 +230,8 @@ def _fn(obj):
             f'_D_{basename}_{partition.fullname.replace(":", "_")}',
```

Comment:
I agree that it looks better but the underscore in the beginning was meant to not clash with user's tests, so that it will not overwrite the directories etc in stage/output. To be honest I don't expect normal tests to finish with `_D_` so I think it's fine to do as you suggested.